src/app/routes/ssoRoutes.js

```diff
@@ -483,6 +487,22 @@ const getQueryParams = (queryObj) => {
     .join('&');
 }
 
+/**
+ * To generate session for state user logins
+ * using server's time as iat and exp time as 5 min
+ * Session will not be created if exp is expired
+ * @param data object to encrypt data
+ * @returns {string}
+ */
+const encryptUserData = (data) => {
```

Comment:
@deveshMantra - we are doing more than just encrypting the data in this method. why are we using JWT format - is it just to reuse verifyToken method or is there any other benefit? 
@vinukumar-vs